pkgs/applications/graphics/inkscape/default.nix

```diff
@@ -40,13 +41,14 @@ stdenv.mkDerivation rec {
       --replace '"python-interpreter", "python"' '"python-interpreter", "${python2Env}/bin/python"'
   '';
 
-  nativeBuildInputs = [ pkgconfig cmake makeWrapper python2Env ]
+  nativeBuildInputs = [ pkgconfig cmake makeWrapper python2Env wrapGAppsHook ]
     ++ (with perlPackages; [ perl XMLParser ]);
   buildInputs = [
     libXft libpng zlib popt boehmgc
     libxml2 libxslt glib gtkmm2 glibmm libsigcxx lcms boost gettext
     gsl poppler imagemagick libwpg librevenge
     libvisio libcdr libexif potrace hicolor-icon-theme
+    gdk-pixbuf librsvg
```

Comment:
Could you remove `gdk-pixbuf` and add explanation comment for parity with Inkscape 1.0?

https://github.com/NixOS/nixpkgs/pull/50286/files#diff-4b701efaede08d4c9cb9f9a3be9154e4R124